sqlglot/expressions.py

```diff
@@ -1094,6 +1094,7 @@ class Insert(Expression):
         "overwrite": False,
         "exists": False,
         "partition": False,
+        "or_": False,
```

Comment:
we can just call it "or"

Reply:
I don't think we can, it clashes with python's `or` when we try to instantiate an `exp.Insert` object using kwargs.